pydatalab/src/pydatalab/routes/v0_1/info.py

```diff
@@ -143,3 +144,50 @@ def list_block_types():
             ).json()
         )
     )
+
+
+def get_all_items_models():
+    return Item.__subclasses__()
+
+
+@INFO.route("/info/types", methods=["GET"])
+def list_supported_types():
+    """Returns a list of supported schemas."""
+    types = [cls.schema()["properties"]["type"]["default"] for cls in get_all_items_models()]
+    types.append(Collection.schema()["properties"]["type"]["default"])
+
+    return jsonify(types)
```

Comment:
```suggestion
    return jsonify(types)
```

This should also be encapsulated in a JSON:API response under a `data` key.